pkg/manager/options.go

```diff
@@ -90,7 +90,7 @@ type Options struct {
 }
 
 func (o *Options) defaults() {
-	if o.Logger == nil {
+	if o.Logger.GetSink() == nil {
```

Comment:
We can no longer do a nil check on `Logger`. Looking at random projects which use `go-logr`, people seem to be using `GetSink()` as a way to check if a logger has been initialized, so I've done the same.